cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -241,18 +246,10 @@ class EvaluatorCommand : OrtCommand(
             }
         }
 
-        val script = when (rules) {
-            is FileType -> (rules as FileType).file.readText()
-
-            is StringType -> {
-                val rulesResource = (rules as StringType).string
-                javaClass.getResource(rulesResource)?.readText()
-                    ?: throw UsageError("Invalid rules resource '$rulesResource'.")
-            }
-        }
-
         if (checkSyntax) {
-            if (Evaluator().checkSyntax(script)) {
+            val evaluator = Evaluator()
+
+            if (scripts.all { evaluator.checkSyntax(it) }) {
```

Comment:
It would be nice if the "Syntax check succeeded/failed" logs would also print which resource they refer to. E.g. `script` could be an object that contains not only the text but also the path of the resource, then this information would be available here. Otherwise it could be difficult to find out which file needs to be fixed.
The output could look like:
```
Syntax check for 'path1' succeeded.
Syntax check for 'path2' failed.
Syntax check for 'path1' succeeded.
```